src/applications/ivc-champva/10-7959a/config/form.js

```diff
@@ -43,6 +53,8 @@ const formConfig = {
         page1: {
           path: 'first-page',
           title: 'First Page',
+          // Placeholder data so that we display "beneficiary" in title when `fnp` is used
+          initialData: { applicantName: { first: 'Beneficiary' } },
```

Comment:
This will be removed when the applicant information section is actually added to the form.